xilem/src/view/sized_box.rs

```diff
@@ -3,7 +3,7 @@
 
 use std::marker::PhantomData;
 
-use masonry::widget;
+use masonry::widget::{self, Padding};
```

Comment:
```suggestion
use masonry::widget;
pub use masonry::widget::Padding;
```
(possibly with a different ordering of the items)

Reply:
Accepted